packages/components/src/index.ts

```diff
@@ -1 +1 @@
-export * from "./components/Sample";
+export * from "./primitives/Box";
```

Comment:
Why are we using the `primitives` concept here? What components should be placed in this folder, for example? 🤔

Reply:
Primitives are the building block components we can use to build other components. Things like Box and Text. We could also have an un-styled collapse, menu, and tab primitives. Those would come with the functionality but not styles. Some of these may just be exported headless ui components.